prueba/src/anagram/AnagramIdentifier.java

```diff
@@ -0,0 +1,29 @@
+package anagram;
+
+public class AnagramIdentifier {
+
+	public boolean anagramDetect(String word1, String word2) {
+
+		if (word1.length() == word2.length()) {
+
+			int j;
+			int countEqualLetters = 0;
+
+			for (int i = 0; i < word1.length() - 1; i++) {
+				j = word2.length() - i - 1;
+				if (word1.charAt(i) == word2.charAt(j)) {
+					countEqualLetters++;
+				}
+			}
+			if ((countEqualLetters == word1.length() - 1)) {
+				return true;
```

Comment:
Remember what I told you. As you are returning a value here, there's no need to handle an `else` case.
This code does the same:
```java
if ((countEqualLetters == word1.length() - 1)) {
    return true;
}

return false;
```

You get it?